tox.ini

```
@@ -21,11 +25,9 @@ commands = pytest -v --tb native -s {posargs} {toxinidir}/tests/unit
 
 [testenv:integration]
 deps =
-    # Until 2.8.6 is released
     https://github.com/juju/python-libjuju/archive/master.zip#egg=juju
     pytest
-    git+https://github.com/charmed-kubernetes/pytest-operator.git@loop
-    # pytest-operator 
+    git+https://github.com/charmed-kubernetes/pytest-operator
```

This should also be switched to PyPI.

```suggestion
    pytest-operator
```